src/test/java/homework_1/tests/Constants.java

```diff
@@ -0,0 +1,9 @@
+package homework_1.tests;
+
+/**
+ * Created by dinar on 14.11.2019.
+ */
+public class Constants {
+    public static final long LONG_MAX = 9223372036854775807l;
```

Comment:
Do you really need these constants? 
Please look to the Java doc if you need some specific values for the primitives wrappers (https://docs.oracle.com/javase/8/docs/api/)